examples/ruby/run

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash -e
```

Comment:
I think much of the work in this script could go in a `Dockerfile`, thereafter we run the Ruby example inside of it. That way, the user doesn't have to install any dependencies and we can control which versions of ruby, rvm, etc. are being used.